corehq/preindex/migrations/0002_initial.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.20 on 2023-07-12 20:09
+
+import corehq.preindex.django_migrations
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    initial = True
```

Comment:
@millerdev is this supposed to have `initial = True` for every time you run `./manage.py makemigrations preindex` as per https://github.com/dimagi/commcare-hq/blob/master/CONTRIBUTING.rst#reindex--migration?

Reply:
Probably not. I'm not sure why it's doing that.

Reply:
Is this line something I should remove from this PR? Perhaps opening a separate PR to address this afterwards

Reply:
Typically each new migration depends on the previous one, and it's usually automatically generated
```suggestion
    dependencies = [
        ('preindex', '0001_initial'),
    ]
```
It's not clear why that didn't happen here. There's probably no harm in fixing it.